### ISSUE TITLE
How to use lsp?

### ISSUE THREAD

**Issue by qqhgs** [ChristianChiarulli/LunarVim#608]
I don't know how to use lsp properly. Now this is what i do.
1. Add new file on lua/lsp. ex. lua-ls.lua
2. Add new line on init.lua. ex.  if O.lang.lua.active then require('lsp.lua-ls') end
3. Add new line on lv-config.lua. ex. O.lang.lua.active = true
This is work but im not sure if i do the right way?
Im not found any doc for that here.

**Comment by 3301n** [comment]
What do you mean by using lsp?

**Comment by ChristianChiarulli** [comment]
first I recommend installing the new stable config.

**Comment by Uzaaft** [comment]
I just migrated from an older version to the new stable config. Love the work! But, it's unclear how one should go ahead and add a lsp(In my case, how to use rust-analyzer with lunar vim). There are no docs on it(besides the LspInstall command), nor any comments(that I can find) in the code.  

Previously, all I had to do is to set the variable for rust_analyzer to true, install it, and done. But after migrating to the new config, this did not work.  

I understand that you guys have a lot to do, but it would be nice from a user standpoint to get some better docs. If there are anyone who can point me in the right position and guide me, I might attempt at helping out regarding the docs, given that I have time.  

Keep up the great work!
